docs/getting-started/migrating-workflows-to-github-apps.rst

```diff
@@ -0,0 +1,120 @@
+Migrating Your Existing Workflows to Use GitHub Apps
+======================================================
+
+Dockstore 1.9.0 provides users with a way to keep their workflows automatically updated (instead of needing to manually refresh) by using GitHub apps.
+Here, we will go over how to migrate your existing Dockstore workflows to use GitHub apps. Also keep in mind that the migration process is done one branch/version at a time.
```

Comment:
It both is and isn't exactly one at a time. The user might think this means they must migrate each version, but they do not have to.

If the user has a workflow with multiple branches, then adds the dockstore.yml to branch foobar, then pushes foobar, the whole workflow becomes dockstore.yml. Any new versions with a .dockstore.yml will be added. Existing versions will persist, the user can even individually refresh them. If they add a .dockstore.yml to one of these existing versions, it will change into a .dockstore.yml version (meaning it is automatically pulled).

Reply:
I decided to delete the sentence and added a new sentence towards the end. Idk if that made this better lol